### PR TITLE
Introduces a keras_recipe on using the new Conv.convolution_op() API

### PR DESCRIPTION
**Pull request by LukeWood**
This goes over how to use the feature I added a month~ or so back.  I think it's really nifty and makes implementing custom convolutions (i.e. StandardizedConv2D) much cleaner.  Let me know what you think!

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! I'll follow up with a round of copyedits :)

File: examples/keras_recipes/subclassing_conv_layers.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+"""
+Title: Subclassing Conv Layers with Conv.convolution_op()
```
[the remark this answers is not in the document]
Comment (LukeWood) [reply examples/keras_recipes/subclassing_conv_layers.py]: Done

File: examples/keras_recipes/subclassing_conv_layers.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Author: [lukewood](https://lukewood.xyz)
+Date created: 11/03/2021
+Last modified: 11/03/2021
+Description: This example shows how to implement custom convolutions using the new Conv.convolution_op() api.
```
[the remark this answers is not in the document]
Comment (LukeWood) [reply examples/keras_recipes/subclassing_conv_layers.py]: Done

File: examples/keras_recipes/subclassing_conv_layers.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import tensorflow.keras as keras
+
+print(keras.__version__)
+"""
+If you are not using Keras 2.7.0 or above, you can install it by running the following
+commands:
+```bash
+pip uninstall tensorflow -y
+pip install tensorflow==2.7.0
+```
```
[the remark this answers is not in the document]
Comment (LukeWood) [reply examples/keras_recipes/subclassing_conv_layers.py]: Removed

File: examples/keras_recipes/subclassing_conv_layers.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+import tensorflow as tf
+import tensorflow.keras as keras
+import keras.layers as layers
+import numpy as np
+
+
+class StandardizedConv2DWithOverride(layers.Conv2D):
+    def convolution_op(self, inputs, kernel):
+        mean, var = tf.nn.moments(kernel, axes=[0, 1, 2], keepdims=True)
+        return super(StandardizedConv2DWithOverride, self).convolution_op(
```
[the remark this answers is not in the document]
Comment (LukeWood) [reply examples/keras_recipes/subclassing_conv_layers.py]: Done!  Good idea

**Review by fchollet — COMMENTED** [review]
Thanks for the update! I pushed some copyedits just now. Please add the generated files.

**Comment by LukeWood** [comment]
Wow - apologies I accidentally force pushed this

**Comment by LukeWood** [comment]
I was attempting to push to my fork

**Pull request closed without merging**